tests/test_response.py

```diff
@@ -122,6 +122,24 @@ class RandomException(LambdaFWException):
             "statusCode": 500,
         }
 
+    def test__from_exception__adds_extra_data_to_response_body_if_only_declared(self) -> None:
+        class RandomException(LambdaFWException):
+            error_code = "RAND001"
+
+        response = Response.from_exception(RandomException(extra={"extra": "random"}), "req-id")
```

Comment:
The same here, do not use a key named `extra` as it may be wrongly interpreted during reading the results of this test.